libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.h

```diff
@@ -67,6 +67,7 @@ class WiFiClientSecure : public WiFiClient {
       _knownkey_usages = usages;
     }
     // Only check SHA1 fingerprint of certificate
+    void setFingerprint(const String& fingerprint);
```

Comment:
Same comment as above about making this eat `char *` instead. Strings on this machine are evil and beat up the heap like mad.  IMO they need to be rewritten to use handles like old MacOS to be easier w/o MMU, but that's another can of worms.